libs/pika/async_mpi/src/mpi_polling.cpp

```diff
@@ -124,20 +185,51 @@ namespace pika::mpi::experimental {
         /// a single instance of all the mpi variables initialized once at startup
         static mpi_data mpi_data_;
 
+        // default transfer mode for mpi continuations
+        static std::size_t task_completion_mode_ = get_completion_mode_default();
+
+        static std::string polling_pool_name = "pika:polling";
```

Comment:
the "pika" in the name is so that when users dump out thread bindings, they see that this was a pool created by pika, and not one of their own pools (if they created some), but to be fair, it's fine without the pika and I'll get rid of it.